api/internal/managers/linux/installation/config.go

```diff
@@ -233,3 +235,55 @@ func (m *installationManager) ConfigureHost(ctx context.Context, rc runtimeconfi
 
 	return nil
 }
+
+// CalculateRegistrySettings calculates registry settings for airgap installations
+func (m *installationManager) CalculateRegistrySettings(ctx context.Context, rc runtimeconfig.RuntimeConfig) (*types.RegistrySettings, error) {
+	// Only return registry settings for airgap installations
+	if m.airgapBundle == "" {
+		return nil, nil
+	}
+
+	// Get the current config to access service CIDR
```

Comment:
looking at the codebase, i think you should only use the runtimeconfig. the installation config is only used for the setup form and accepting values from the user, which then gets used to calculate and update the runtimeconfig. the runtimeconfig is the source of truth